src/api/app/assets/javascripts/webui2/package-view_file.js

```diff
@@ -0,0 +1,30 @@
+var DAYS = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
+var MONTHS = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'];
+
+// The jshint ignore:line is needed to ignore the offense: `'addChangesEntryTemplate' is defined but never used.` This is used in the view directly
+function addChangesEntryTemplate() { // jshint ignore:line
+  var date = new Date(),
+      day = date.getUTCDate().toString(),
+      hours,
+      minutes,
+      seconds,
+      template;
+
+  if (day.length === 1) { day = " " + day; } // Pad single character day value
+  hours = date.getUTCHours();
+  if (hours < 10) { hours = '0' + hours; }
```

Comment:
This could be simplified :see_no_evil: but I guess it is out of scope of the PR, I'll send a new one. :wink:

Reply:
Yes, I know the code is quite bad... I didn't feel like refactoring this. I will add a `FIXME: Refactor this function`

Reply:
not needed, I'll send a PR as soon as this is merged! :wink: